downstream/modules/platform/ref-hub-variables.adoc

```diff
@@ -85,26 +101,100 @@ Behavior is driven by `galaxy-importer.cfg` configuration.
 
-When the bundle installer is run, by a new repository is created by default in {PrivateHubName} named `validated`` and the list of the validated collections is updated.
+For example, \https://<load balancer host>.
 
-If you do not want to install this content, there are two options:
+If not specified, the first node in the `[automationhub]` group is used.
 
-Set `automationhub_seed_collections` to `false`.
+Use `automationhub_main_url` to specify the main {HubName} URL that clients connect to if you are implementing {RHSSO} on your {HubName} environment.
+| *`automationhub_pg_database`* | _Required_
 
-Set `automationhub_seed_collections` to `true` and `automationhub_collection_seed_repository` to `certified`.
+The database name.
 
+Default = `automationhub`
+| *`automationhub_pg_host`* | Required if not using internal database.
+| *`automationhub_pg_password`* | The password for the {HubName} PostgreSQL database.
 
-Set to `False` to disable the seeding.
+Do not use special characters for `automationhub_pg_password`. 
+They can cause the password to fail.
+| *`automationhub_pg_port`* | Required if not using internal database.
+
+Default = 5432
+| *`automationhub_pg_sslmode`* | Required.
+
+Default = `prefer`
+| *`automationhub_pg_username`* | Required 
+
+Default = `automationhub`
+| *`automationhub_require_content_approval`* | _Optional_
+
+If {HubName} enforces the approval mechanism before collections are made available.
+
+By default when you upload collections to {HubName} an administrator must approve it before it is made available to the users. 
+
+If you want to disable the content approval flow, set the variable to `false`.
 
 Default = `true`
-| *`automationhub_ui_download_url`* | Set URL to acquire the {HubName} UI for internal testing.
+| *`automationhub_seed_collections`* | A boolean that defines whether or not preloading is enabled.
+
+When the bundle installer is run, validated content is uploaded to the `validated` repository, and certified content is uploaded to the `rh-certified` repository.
+
+By default, both certified and validated content are uploaded. 
+
+If you do not want to install one type of content, there are two options:
+
+* Set `automationhub_seed_collections` to `false` o disable the seeding, or
+
+* Set `automationhub_seed_collections` to `true` and `automationhub_collection_seed_repository` to the type of content you do want to include.
+
+Default = `true`.
+| *`automationhub_ssl_cert`* | _Optional_ 
+
+`/path/to/automationhub.cert`
+Same as `web_server_ssl_cert` but for {HubName} UI and API
+| *`automationhub_ssl_key`* | _Optional_
+
+`/path/to/automationhub.key`
+
+Same as `web_server_ssl_key` but for {HubName} UI and API
+| *`automationhub_ssl_validate_certs`* | For {PlatformName} 2.2 and later, this value is no longer used.
+
+If {HubName} should validate certificate when requesting itself because by default, {PlatformNameShort} deploys with self-signed certificates.
```

Comment:
```suggestion
Value is `true` if {HubName} should validate certificate when requesting itself because by default, {PlatformNameShort} deploys with self-signed certificates.
```